backend/server/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+
+# Path to default image
+DEFAULT_IMAGE = '../../images/app_logo.png'
+pic_folder = './migrations/pic_folder'
+
```

Comment:
In case of photo, I think it would be nice to give a default photo, for the sake of completeness, rather than just leaving it blank. But I will modify it for now.